ros/openapi/openapi.json

```diff
@@ -104,6 +104,35 @@
                 }
             }
         },
+        "/systems/{inventory_id}/recommendations": {
+            "get": {
+                "summary": "Show the list of recommendations by given inventory id",
+                "description": "Show the list of recommendations by given inventory id",
+                "parameters": [
```

Comment:
I think here you can also add information about `description` filter?

Reply:
Done